tests/screenshot/apps/components/Scroller.js

```diff
@@ -1,12 +1,86 @@
+import Button from '../../../../Button';
+import img from '../../images/600x600.png';
+import ImageItem from '../../../../ImageItem';
 import Scroller from '../../../../Scroller';
+import ri from '@enact/ui/resolution';
 
 import css from './Scroller.module.less';
 
+const imageItems = [];
+const defaultdataSize = 3;
+
+const renderImageItem = (dataSize) => {
+
+	imageItems.length = 0;
+
+	for (let i = 0; i < dataSize; i++) {
+		const index = i;
+		const source = {img};
+		imageItems.push({source, index});
+	}
```

Comment:
If we use same "img", we don't need imageItems obj

Please think more to make simple code

imageItems = Array.from({length: dataSize}, (v, i) => i); ? 
I'm not sure.

Reply:
Suggested Change.